### DISCUSSION TITLE
從遊戲中反思AI的社會問題

### DISCUSSION THREAD

**Discussion by Jerry10413** [1101-GS4719-C/Python-Community-GS4719-C#983] [General]
分享個很不錯的劇情遊戲:底特律 變人 這是一個跟AI有關的故事，而且我也覺得這個東西在未來也會有可能進入我們的生活，這個遊戲主要是在探討創造出人造生命後(AI)他們是否擁有人權(因為劇情設定AI是用來服務人類的)裡面有一個很有趣的設計，就是在AI受到精神上極大的刺激的時候，就會覺醒自我意識，從而變成擁有自己想法的人，裡面的故事就是在探討有關於各種覺醒後的AI反抗人類的制度的故事，並且根據玩家的選擇會有很多不同的結局，個人覺得是一個非常好的遊戲~~
有興趣可以去看看相關資訊
https://zh.wikipedia.org/zh-tw/%E5%BA%95%E7%89%B9%E5%BE%8B%EF%BC%9A%E8%AE%8A%E4%BA%BA
我個人是蠻推薦的，值得這個價錢(最近剛特價完QQ)

**Comment by wanderer657** [comment]
我覺得《底特律：變人》這款遊戲挺酷的，它主要探討了人工智慧對社會的影響。遊戲中有些暴力場面和粗話可能有點過頭，但我認為這正是為了呈現現實社會中存在的問題，比如種族歧視和社會不公。我發現玩這遊戲能讓你思考很多道德和自由意志等議題，也讓你更深入地了解人工智慧對未來的影響。相較於其他遊戲，這款更注重故事性和劇情，給玩家提供了一個思考人性和道德問題的機會。
